filebeat/fileset/modules.go

```diff
@@ -417,6 +422,11 @@ func (reg *ModuleRegistry) LoadML(esClient PipelineLoader) error {
 
 // SetupML sets up the machine-learning configurations into Elasticsearch using Kibana, if X-Pack is available
 func (reg *ModuleRegistry) SetupML(esClient PipelineLoader, kibanaClient *kibana.Client) error {
+	if !mlimporter.IsCompatible(esClient) {
+		logp.Info("Skipping loading machine learning jobs because of Elasticsearch version is too new.\nIt must be 7.x for setting up it using Beats. Please use the Machine Learning UI in Kibana.")
```

Comment:
Looks like the exact same error message as above but I assume the two functions do something slightly different based on the description? Same question for error instead of log.

Reply:
Yes, this function checks the ES version and the uses the Kibana client to load the ML jobs if it already has the required API.